src/components/Tweet.js

```diff
@@ -1,39 +1,26 @@
-function Tweet() {
+import ProfileImage from '../components/ProfileImage'
+import User from '../components/User'
+import Timestamp from '../components/Timestamp'
+import Message from '../components/Message'
+import Actions from './Actions'
+
+const Tweet = ({name, image, handle, timestamp, message}) => {
   return (
     <div className="tweet">
-      <img
-        src="https://i.imgur.com/9yw1Fyw.jpg"
-        className="profile"
-        alt="profile"
+      <ProfileImage image= {image}
       />
```

Comment:
If you pass full object from App.js, this could be possible destructuring/syntax
```
const Tweet = (props) => {
  const {user, timestamp, message} = props.tweet;
  return (
    <div className="tweet">
      <ProfileImage image= {user.image}
      />
```